README.md

```diff
@@ -153,6 +155,9 @@ The repo contains the following components. More libraries are coming soon. ['Wa
 * **System.Reflection.Metadata**. Provides a highly-tuned, low-level ECMA-335 metadata reader.  This is the same
   reader used by "[Roslyn]" C# and Visual Basic compilers to parse assemblies.
 
+* **System.Reflection.TypeExtensions**. Provides extension methods for System.Type that are designed to be 
```

Comment:
We could say for System.Reflection types.  I just copied the description from the NuGet package.